수박수박수박수.js

```diff
@@ -0,0 +1,22 @@
+// 길이가 n 
+//문자열을 리턴하는 함수
+//n이 4면 수박수박 3이면 수박수
+function solution(n) {
+    //let 과 var의 차이점 (let은 중복 선언이 불가능 하고 var은 중복 선언이 가능하다. )
+    //결과값은 한번만 사용해도 괜찮으니까 let을 사용 한다. 
+        let answer = ""
+        
+    //for 문을 이용해서 문제 풀이 하는 방법
+        for (let i = 1; i<=n; i++ ) {// 선언과 동시에 값을 초기화 해준다.  
+            
+            if (i % 2 ===0){//==은 값만 같으면 true, ===값과 값의 종류가 모두 같아야 true  
+                answer +="박"
+            }
+            else{
+                answer +="수"
+            }
```

Comment:
위와 같이 반복문의 시작을 0부터로 바꿨으니 if문도 달라져야겠죠?
(i % 2 === 0)이면 boolean에서는 false를 의미, (i % 2 === 1)은 true를 의미합니다.

반복문의 시작을 0으로 바꿨으니 (i % 2 === 1)이 들어가야 하는 자리이므로 (i % 2)로 변환이 가능합니다.
추가적으로 삼항연산자까지 사용해 준다면 더 간결한 코드가 되겠죠?
```suggestion
        (i % 2) ? answer += '박' : answer += '수';
```